### ISSUE TITLE
ziggeoReInitApp is not a function

### ISSUE THREAD

**Issue by azanoni** [Ziggeo/ZiggeoWordpressComments#13]
Hi,

We just discovered a frontend "error", when we use the Fluent custom field Video Recorder

![image](https://user-images.githubusercontent.com/21685626/222767384-48b34543-e61e-4a6c-a0d4-111634195fdd.png)

wp-content\plugins\ziggeo\core\assets.php line 210

![image](https://user-images.githubusercontent.com/21685626/222766207-2a66be9b-ee36-4b25-bfef-3980ceb6390e.png)

Maybe is a "de-sync" issue, and we dont know if this is mandatory for any feature of the plugin.

Also I just wanna mention that the changelog of the plugin refers to the version "= 2.14.1 =" when the plugin in this repo is under 3.X and the official repo under WP is 2.12, would be better if we could have a best sync between both repos and changelog.

Thanks in advance

**Comment by Bane-D** [comment]
Thank you for raising this issue Alfredo.

**ziggeoReInitApp**

This function is needed when the lazyload feature is turned on (core plugin, under Settings > Experts tab). It would be interesting to know if you have the lazy load option turned on or off when you get this error?

While we are looking to fix any error with this one if you see the embedding, then it does not matter if there is error.

* Adding it to my todo list to check and see however I did not experience it earlier. Will follow up as I am able to check it further, hopefully over the weekend.

**Changelog**

The changelog file here: https://github.com/Ziggeo/ziggeo-wordpress-plugin/blob/master/CHANGELOG.md is thought of as a log of previous changes. It is updated every time we make the update in the version of the WP that is pushed out.

The latest changes are always in the readme file under the Update Notice: https://github.com/Ziggeo/ziggeo-wordpress-plugin/blob/master/readme.txt#L309

The WordPress repository however might not be updated as often as the Github version is (something we want to change) so the changelog there will be a bit more outdated - at least at this time.

**Comment by Bane-D** [comment]
It should be fixed with the latest update.

https://github.com/Ziggeo/ziggeo-wordpress-plugin/commit/dabcaa02079b03e797c2c872ce63faa548c671f4

**Comment by azanoni** [comment]
Hi, 

Thanks for fix this so quickly, I just updated both plugins,  we have the lazy load off
![image](https://user-images.githubusercontent.com/21685626/223119243-f906e46e-40b6-417c-8e55-80da86a3453b.png)

the issue was fixed but we are seeing a new one, shall I create a new ticket? or is better for you update this one?:

![image](https://user-images.githubusercontent.com/21685626/223120525-51704c7d-75db-4f1c-ad88-5f4188169fa3.png)

![image](https://user-images.githubusercontent.com/21685626/223120647-ffde8b54-e3d2-4be4-9f3d-f83b9fb152c2.png)

**Comment by Bane-D** [comment]
Hi Alfredo, great to hear that original issue was fixed, having mixed feelings about there being another one (happy that you have reported it) :)

In my tests before the push I did not see any errors so I will likely need to play around to recreate it. Will check it out, most probably during the weekend and push the update.

In terms of issues, I appreciate thinking of creating new issue, that would be ideal, however as this is result of update for this one, let's leave it here for now.

PS: For anyone that finds this to affect them as well, a "hack" to fix this would be just changing this line:
https://github.com/Ziggeo/ziggeo-wordpress-plugin/blob/master/core/assets.php#L185 to 
```
'for(i = 0, c = ((typeof ZiggeoWP.lazyload !==\'undefined\') ? ZiggeoWP.lazyload.length : 0); i < c; i++) {'
```

By a quick look I think this should be enough when lazy load is disabled, however I did not have time to test it out properly.
